### DISCUSSION TITLE
AttributeError: module 'ezkl' has no attribute 'export'

### DISCUSSION THREAD

**Discussion by only4sim** [zkonduit/ezkl#366] [Q&A] [answered]
I am trying the official colab notebook https://colab.research.google.com/drive/1XuXNKqH7axOelZXyU3gpoTOCvFetIsKu?usp=sharing. When I was using the ezkl.export(circuit, input_shape = [20]), it gives me this error. The full code is shown as following:
```
!pip uninstall -y ezkl_lib
!pip uninstall -y ezkl
!pip install ezkl_lib
!pip install ezkl

import os
import ezkl

params_path = os.path.join('kzg.params')

res = ezkl.gen_srs(params_path, 17)

from torch import nn

import os
import json


class MyModel(nn.Module):
    def __init__(self):
        super(MyModel, self).__init__()
        self.relu = nn.ReLU()
        self.d2 = nn.Linear(20, 20)


    def forward(self, x):
        x = self.d2(x)
        return self.relu(x)


circuit = MyModel()
ezkl.export(circuit, input_shape = [20])
```

I have checked the [pyezkl](https://github.com/zkonduit/pyezkl), and everything looks fine, so I am really confused.

**Comment by alexander-camuto** [comment]
Hey @only4sim thank you for flagging this. @JSeam2 or @jasonmorton can we update this colab notebook to reflect the new ezkl bindings.

**Reply by jasonmorton** [reply, marked as answer]
This is fixed, thanks @only4sim

**Reply by only4sim** [reply]
Hey @alexander-camuto and @jasonmorton. Thanks for your prompt help! It's amazing how you guys solved the problem in such a short period of time. I just tested it and it looks like everything is running great. 

BTW, I love the lyrics you guys quoted. We got EZKL, we got the torch, what you want :)
